spec/restful_resource/http_client_spec.rb

```diff
@@ -161,7 +161,17 @@ def http_client(connection)
           stubs.get('http://httpbin.org/bearer', 'Authorization' => 'Bearer abc123') { |_env| [200, {}, nil] }
         end
 
-        response = http_client(connection).get('http://httpbin.org/bearer', headers: { 'Authorization' => 'Bearer abc123' })
+        response = http_client_with_auth(connection).get('http://httpbin.org/bearer')
```

Comment:
The initial implementation of the test was a bit odd..
We were setting a custom header in the get request, but we were also setting an auth_token in the 'new' above:
https://github.com/carwow/restful_resource/pull/174/files#diff-c3b6bd2a8913d431903e4421957e8987916ff3137bdb26d37c8c82431281db7dR156

I removed the header in the get request to ensure we weren't testing 2 things